packages/tools/src/utilities/rectangleROITool/isAxisAlignedRectangle.ts

```diff
@@ -0,0 +1,62 @@
+import { vec3 } from 'gl-matrix';
+import { utilities as csUtils } from '@cornerstonejs/core';
+
+/**
+ * Determines whether a given rectangle in a 3D space (defined by its corner
+ * points in IJK coordinates) is aligned with the IJK axes.
+ * @param rectangleCornersIJK - The corner points of the rectangle in IJK coordinates
+ * @returns True if the rectangle is aligned with the IJK axes, false otherwise
+ */
+function isAxisAlignedRectangle(rectangleCornersIJK) {
+  const iAxis = [1, 0, 0];
+  const jAxis = [0, 1, 0];
+  const kAxis = [0, 0, 1];
+
+  function calculateVector(pointA, pointB) {
+    return [
```

Comment:
Why not vec3.sub(vec3.create(),pointB, pointA))?
You might as well use a native array type for the values here.